modules/op-installing-tekton-hub-with-login-and-rating.adoc

```diff
@@ -70,6 +73,42 @@ stringData:
 You can delete the unused fields for the Git repository hosting service providers that are irrelevant to your deployment.
 ====
 
+. Optional: Verify that the {tekton-hub} API config map available in the file `<tekton_hub_root>/config/02-api/21-api-configmap.yaml` is similar to the following example:
++
+.Example: Tekton Hub API config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-hub-api
+  labels:
+    app: tekton-hub-api
+data:
+  CATALOG_REFRESH_INTERVAL: 30m 
+  CATEGORIES: |
+    - Automation
+    - Build Tools
+  CATALOGS: | 
+    - name: tekton
+      org: tektoncd
+      type: community
+      provider: github
+      url: https://github.com/tektoncd/catalog
+      revision: main
+  SCOPES: | 
+    - name: agent:create
+      users: [abc,qwe,pqr]
+    - name: catalog:refresh
+      users: [abc,qwe,pqr]
+    - name: config:refresh
+      users: [abc,qwe,pqr]
+  DEFAULT: | 
+    scopes:
+      - rating:read
+      - rating:write
+---- 
```

Comment:
We can remove this part, user don't need to verify the api config map, if he wants to use his custom data he can directly edit the Hub CR